lib/WP_Auth0_Api_Operations.php

```diff
@@ -99,34 +95,10 @@ public function create_wordpress_connection( $app_token, $migration_enabled, $pa
 		$response = WP_Auth0_Api_Client::create_connection( $domain, $app_token, $body );
 
 		if ( $response === false ) {
-
 			return false;
-
-		}
-
-		$connections   = WP_Auth0_Api_Client::search_connection( $domain, $app_token, 'auth0' );
-		$db_connection = null;
-
-		$migration_connection_id = $response->id;
-
-		foreach ( $connections as $connection ) {
```

Comment:
This method is only called in one place and the calling method already does this check.